common/types/encoding/codecv1/codecv1.go

```diff
@@ -10,6 +10,7 @@ import (
 	"math/big"
 	"strings"
 
+	"github.com/colinlyguo/zstd"
```

Comment:
Do you plan to change this before merging this PR?

Reply:
Yes.